docs/tileentities/tileentity.md

```diff
@@ -0,0 +1,112 @@
+# TileEntities
+
+This is often used by vanilla Minecraft to handle inventories on chests, smelting logic on furances, or area effects for beacons. 
+More advanced versions exist in mods such as quaries, sorting machines, pipes, and displays
+!!! note
+	`TileEntities` aren't a solution for everything, their loading distance is limited, and they can cause lag when used in large amounts.
+	When possible, try to avoid them.
+
+## Creating a `TileEntity`
+In order to create a `TileEntity` you need to extend the `TileEntity` class.
+It is important that your `TileEntity` has a default constructor, so that Minecraft can properly load it.
+After you've created your class, you need to register the `TileEntity`. For this you need to call 
+```JAVA
+	GamerRegistry#registerTileEntity(Class<? extends TileEntity> tileEntityClass, ResourceLocation key)
+```
+Where the first parameter is simply your TileEntity class and the second the registry name of your TileEntity.
+
+!!! note
+	The method to register a `TileEntity` is using a `String` instead of a `ResourceLocation` before Forge version `14.23.3.2694`
+	You should still use the `ResourceLocation` format `modid:tileentity` though.
```

Comment:
This method creates a ResourceLocation from the String. Be sure to use the `ResourceLocation` format `modid:tile_entity` to avoid it being called `minecraft:tile_entity`.